src/App.tsx

```diff
@@ -1,41 +1,40 @@
-import "./App.css";
-import Fold from "./components/layout/Fold";
-import Footer from "./components/layout/footer/Footer";
-import HamburgerMenu from "./components/layout/navbar/hamburgerMenu/HamburgerMenu";
-import Navbar from "./components/layout/navbar/Navbar";
-import { WrapperMain } from "./components/layout/wrapper/Wrapper.styled";
-import Contact from "./sections/contact/Contact";
-import Events from "./sections/events/Events";
-import Intro from "./sections/intro/Intro";
-import Organisers from "./sections/organisers/Organisers";
-import Partners from "./sections/partners/Partners";
-import Prime from "./sections/prime/Prime";
-import Countdown from "./sections/countdown/Countdown";
-import Program from "./sections/program/Program";
-import Tickets from "./sections/tickets/Tickets";
-import { LteTablet } from "./theme/MediaQueries";
-import Snowing from "./components/snowing/Snowing";
+import './App.css';
+import Fold from './components/layout/Fold';
+import Footer from './components/layout/footer/Footer';
+import HamburgerMenu from './components/layout/navbar/hamburgerMenu/HamburgerMenu';
+import Navbar from './components/layout/navbar/Navbar';
+import { WrapperMain } from './components/layout/wrapper/Wrapper.styled';
+import Contact from './sections/contact/Contact';
+import Events from './sections/events/Events';
+import Intro from './sections/intro/Intro';
+import Organisers from './sections/organisers/Organisers';
+import Partners from './sections/partners/Partners';
+import Prime from './sections/prime/Prime';
+import Countdown from './sections/countdown/Countdown';
+import Program from './sections/program/Program';
+import Tickets from './sections/tickets/Tickets';
+import { LteTablet } from './theme/MediaQueries';
+import Snowing from './components/snowing/Snowing';
+import Consent from './components/consent/Consent';
 
-// import CookieConsent from 'react-cookie-consent';
-
-import reveal from "./functions/reveal";
-window.addEventListener("scroll", reveal);
+import reveal from './functions/reveal';
+window.addEventListener('scroll', reveal);
 
 function App() {
   return (
     <WrapperMain>
-      {/* <CookieConsent /> */}
+      <Consent />
```

Comment:
Maybe better naming, Consent is general and might be used for other actions as well